src/test/java/com/sergii/tests/factory/ProductDetailsPageFactoryTest.java

```diff
@@ -0,0 +1,63 @@
+package com.sergii.tests.factory;
+
+import com.sergii.code.factory.ProductDetailsPageFactory;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import java.time.Duration;
+
+
+public class ProductDetailsPageFactoryTest {
```

Comment:
Pls, solve all of the comments for HomePage tests

Reply:
Done and appropriate comments have been added. Thank you for review.